content/influxdb/v2.0/upgrade/v1-to-v2/docker.md

```diff
@@ -0,0 +1,245 @@
+3.	1.x configuration file mounted at `/etc/influxdb/influxdb.conf`
+4.	1.x data directory mounted at `/var/lib/influxdb`
+
+{{% note %}}
+#### Avoid data loss
+The automated upgrade process generates both data and configuration files, by default,
+under `/var/lib/influxdb2` and `/etc/influxdb2`.
+We recommend mounting volumes at both paths to avoid losing data.
+{{% /note %}}
+
+### Upgrade initialization mode
+Set the `DOCKER_INFLUXDB_INIT_MODE` environment variable to `upgrade`.
+
+## Upgrade examples
+The examples below provide information for different InfluxDB Docker upgrade scenarios.
+
+- [Minimal upgrade](#minimal-upgrade)
+- [Upgrade with a custom InfluxDB 1.x configuration file](#upgrade-with-a-custom-influxdb-1-x-configuration-file)
+- [Upgrade with custom paths](#upgrade-with-custom-paths)
+
+### Minimal upgrade
+If you're currently running a minimal InfluxDB 1.x deployment similar to:
+
+```sh
+docker run -p 8086:8086 \
+  -v influxdb:/var/lib/influxdb \
+  influxdb:1.8
+```
+
+**To upgrade this minimal deployment to InfluxDB 2.x:**
+
+1.  Stop the running InfluxDB 1.x container.
+2.  Start the InfluxDB container and specify the following:
+
+    - Volume mount for the InfluxDB 1.x data directory
+    - Volume mount for the InfluxDB 2.x data directory
+    - InfluxDB [initialization mode](#upgrade-initialization-mode) environment variable
+    - InfluxDB [initialization credential](#influxdb-2x-initialization-credentials) environment variables
+    - `influxdb:2.0` Docker image
+
+    ```sh
+    docker run -p 8086:8086 \
+      -v influxdb:/var/lib/influxdb \
+      -v influxdb2:/var/lib/influxdb2 \
+      -e DOCKER_INFLUXDB_INIT_MODE=upgrade \
+      -e DOCKER_INFLUXDB_INIT_USERNAME=my-user \
+      -e DOCKER_INFLUXDB_INIT_PASSWORD=my-password \
+      -e DOCKER_INFLUXDB_INIT_ORG=my-org \
+      -e DOCKER_INFLUXDB_INIT_BUCKET=my-bucket \
+      influxdb:2.0
+    ```
+
+### Upgrade with a custom InfluxDB 1.x configuration file
+If you're currently running an InfluxDB 1.x deployment with custom configuration
+file similar to:
+
+```sh
+docker run -p 8086:8086 \
+  -v influxdb:/var/lib/influxdb \
+  -v $PWD/influxdb.conf:/etc/influxdb/influxdb.conf:ro \
+  influxdb:1.8
+```
+
+**To upgrade an InfluxDB 1.x deployment with a custom configuration file to InfluxDB 2.x:**
+
+1.  Stop the running InfluxDB 1.x container.
+2.  Start the InfluxDB container and specify the following:
+
+    - Volume mount for the InfluxDB 1.x data directory
+    - Volume mount for the InfluxDB 1.x configuration file
+    - Volume mount for the InfluxDB 2.x data directory
+    - InfluxDB [initialization mode](#upgrade-initialization-mode) environment variable
+    - InfluxDB [initialization credential](#influxdb-2x-initialization-credentials) environment variables
+    - `influxdb:2.0` Docker image
+
+```sh
+docker run -p 8086:8086 \
+  -v influxdb:/var/lib/influxdb \
+  -v $PWD/influxdb.conf:/etc/influxdb/influxdb.conf:ro \
+  -v influxdb2:/var/lib/influxdb2 \
```

Comment:
Also needs a volume mount for `/etc/influxdb2`, to capture the generated config file.